### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by cybercyst** [pydantic/pydantic#3228] [Question] [answered]
I just saw a PR had been landed with work to enable intellisense in VS Code, etc

Is there any chance we'll see a new release of pydantic in the near future? It has been some time since the last one.

Thanks for all the hard work you are doing! It is much appreciated!

**Comment by samuelcolvin** [comment, marked as answer]
Yes I need to make a new release, I'm sorry I haven't done so already.

I have a new baby and extremely demanding job which means I gave less time normal to commit (literally) to open source.

I'll try to get a release out asap but I can't give an exact timeframe.

In the meantime, if you have a minute, please review open prs and comment if you specifically want them including in the upcoming release.

**Reply by lovetoburnswhen** [reply]
congrats!

**Comment by cybercyst** [comment]
@samuelcolvin completely understandable! Maintenance on projects like this can be a thankless job, but I very much appreciate this project and your work in it!

**Comment by iedmrc** [comment]
any update on this?
